### PR TITLE
feat: updates example with clearer naming, groups example, descriptions and typo fixes

### PR DESCRIPTION
**Pull request by tnolet**
…

I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

## Affected Components
* [ ] CLI
* [ ] Create CLI
* [ ] Test
* [x] Docs
* [x] Examples
* [ ] Other

## Notes for the Reviewer
- reshuffles advanced example
- adds a more advanced grouping example
- adds descriptions to `checkly.config.ts`
- adds playwright/test as dependency
- fixes typos

### CODE REVIEW

**Review by hlenke — COMMENTED** [review]
suggest setTimeout by default. Maybe comment it out to.

**Review by hlenke — COMMENTED** [review]
suggest setTimeout by default. Maybe comment it out to.

File: examples/boilerplate-project/__checks__/homepage.spec.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-// @ts-ignore
 import { test, expect } from '@playwright/test'
 
 test('Checkly Homepage', async ({ page }) => {
```
Comment (hlenke) [inline examples/boilerplate-project/__checks__/homepage.spec.ts:3]:
```suggestion
test('Checkly Homepage', async ({ page }) => {
  //override Playwright Test default timeout to 110s
  test.setTimeout(110000);
```

File: examples/advanced-project/src/services/docs/__checks__/docs-search.spec.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { test, expect } from '@playwright/test'
 import { defaults } from '../../../defaults'
 
 /**
- * This spec file will be picked up and turned into a Browser check, using the defaults from the checkly.config.js file
+ * This spec file will be picked up and turned into a Browser Check, using the defaults from the checkly.config.ts file
  * at the root of this example.
  */
```
Comment (hlenke) [inline examples/advanced-project/src/services/docs/__checks__/docs-search.spec.ts:9]: Add the following in the tests below. 
    //override Playwright Test default timeout to 110s
    test.setTimeout(110000);

**Review by hlenke — COMMENTED** [review]
Add overwriting default timeout to: test.setTimeout(110000); as a hint for new users.

File: examples/advanced-project/checkly.config.ts [hunk 3]
```diff
@@ -1,21 +1,42 @@
...
+    */
     runtimeId: '2022.10',
-    checkMatch: '**/*.check.ts',
+    /* A glob pattern that matches the Checks inside your repo, see https://www.checklyhq.com/docs/cli/using-check-test-match/ */
+    checkMatch: '**/__checks__/**/*.check.ts',
     browserChecks: {
-      testMatch: '**/__checks__/*.spec.ts', // this matches any Playwright spec-files and automagically creates a Browser check
+      /* A glob pattern matches any Playwright .spec.ts files and automagically creates a Browser Check. This way, you
+      * can just write native Playwright code. See https://www.checklyhq.com/docs/cli/using-check-test-match/
+      * */
```
[the remark this answers is not in the document]
Comment (tnolet) [reply examples/advanced-project/checkly.config.ts:30]: done

**Review by hlenke — APPROVED** [review]
LGTM

File: examples/advanced-project/README.md [hunk 4]
```diff
@@ -57,6 +59,17 @@ Run the core CLI commands with `npx checkly <command>`
 | `npx checkly login`  | Log in to your Checkly account                   |
 | `npx checkly --help` | Show help for each command.                      |
 
+[Check the docs for the full CLI reference](https://www.checklyhq.com/docs/cli/command-line-reference/)
```
Comment (stefanjudis) [inline examples/advanced-project/README.md:62]:
```suggestion
[Check the docs for the full CLI reference](https://www.checklyhq.com/docs/cli/command-line-reference/).
```

File: examples/advanced-project/README.md [hunk 5]
```diff
@@ -57,6 +59,17 @@ Run the core CLI commands with `npx checkly <command>`
...
+
+## Adding and running `@playwright/test`
+
+You can add `@playwright/test` to this project to get full code completion and run `.spec.ts` files for local debugging.
+It's best to install the Playwright NPM package version that matches your [Checkly runtime](https://www.checklyhq.com/docs/cli/npm-packages/)
+
+```bash
+npm i -D @playwright/test@1.28.0
```
Comment (stefanjudis) [inline examples/advanced-project/README.md:70]:
```suggestion
npm install --save-dep @playwright/test@1.28.0
```

File: examples/advanced-project/README.md [hunk 6]
```diff
@@ -57,6 +59,17 @@ Run the core CLI commands with `npx checkly <command>`
 | `npx checkly login`  | Log in to your Checkly account                   |
 | `npx checkly --help` | Show help for each command.                      |
 
+[Check the docs for the full CLI reference](https://www.checklyhq.com/docs/cli/command-line-reference/)
+
+## Adding and running `@playwright/test`
+
+You can add `@playwright/test` to this project to get full code completion and run `.spec.ts` files for local debugging.
+It's best to install the Playwright NPM package version that matches your [Checkly runtime](https://www.checklyhq.com/docs/cli/npm-packages/)
```
Comment (stefanjudis) [inline examples/advanced-project/README.md:67]:
```suggestion
It's best to install the Playwright npm package version that matches your [Checkly runtime](https://www.checklyhq.com/docs/cli/npm-packages/)
```

File: examples/advanced-project/checkly.config.ts [hunk 7]
```diff
@@ -1,21 +1,44 @@
...
+   * See https://www.checklyhq.com/docs/cli/constructs/ to learn more about logical IDs.
+   */
   logicalId: 'advanced-example-project',
+  /* An optional URL to your Git repo */
   repoUrl: 'https://github.com/checkly/checkly-cli',
+  /* Sets default values for Checks */
   checks: {
+    /* A default for how often your Check should run in minutes */
+    frequency: 10,
+    /* Checkly datacenters where to run your Checks as monitors */
```
Comment (stefanjudis) [inline examples/advanced-project/checkly.config.ts:19]:
```suggestion
    /* Checkly data centers to run your Checks as monitors */
```

**Review by stefanjudis — APPROVED** [review]
Just added some nits here'n'there. Feel free to move forward. :)

**Comment by tnolet** [comment]
@stefanjudis addressed comments. Waiting for a LGTM

**Review by stefanjudis — APPROVED** [review]

**Comment by stefanjudis** [comment]
:shipit:

**Pull request merged**